CHANGELOG.md

```diff
@@ -1,3 +1,39 @@
+## Release 0.7.0
+
+New features:
+- Track short-lived connections
+  [#356](https://github.com/weaveworks/scope/issues/356)
+- Details pane: add a lot more information and present it in a more intuitive way.
+  [#394](https://github.com/weaveworks/scope/pull/394)
+  [#396](https://github.com/weaveworks/scope/pull/396)
+  [#400](https://github.com/weaveworks/scope/pull/400)
+  [#387](https://github.com/weaveworks/scope/pull/387)
+  [#370](https://github.com/weaveworks/scope/issues/370)
+  [#322](https://github.com/weaveworks/scope/issues/322)
+- Make edge direction flow from client to server.
+  [#355](https://github.com/weaveworks/scope/pull/355)
+- Make pseudo nodes clickable.
+  [#249](https://github.com/weaveworks/scope/issues/249)
+  [#401](https://github.com/weaveworks/scope/pull/401)
+  [#426](https://github.com/weaveworks/scope/pull/426)
+- Truncate node names and text in the details pane.
+  [#429](https://github.com/weaveworks/scope/pull/429)
+  [#430](https://github.com/weaveworks/scope/pull/430)
+- Show container name instead of process name in the *Applications* view.
+  [#331](https://github.com/weaveworks/scope/issues/331)
+- Improve graph transitions between updates.
+  [#379](https://github.com/weaveworks/scope/pull/379)
+- Label system containers with `works.weave.role=system`.
+  [#337](https://github.com/weaveworks/scope/issues/337)
+- Support upcoming Weave status interface/
+  [#389](https://github.com/weaveworks/scope/pull/389)
+
+Bug fixes:
+- *Trying to reconnect..* in UI even though its connected.
+  [#392](https://github.com/weaveworks/scope/pull/392)
+- Fix JS error on missing edge target.
+  [#443](https://github.com/weaveworks/scope/pull/443)
```

Comment:
This one hasn't been merged yet, but I've included it already since it's a blocker.